### DISCUSSION TITLE
Look like using TailwindCSS have conflict using react-markdown

### DISCUSSION THREAD

**Discussion by ZanyuanYang** [remarkjs/remark#1191] [Q&A]
How I can fix this?

**Comment by ChristianMurphy** [comment]
Hey @ZanyuanYang! 👋 

I'd highly recommend searching existing discussions, this question has been asked and answered several times.
In particular see:

* https://github.com/remarkjs/react-markdown/issues/505 (most of the rest refer to the answer here)
* https://github.com/orgs/remarkjs/discussions/1142
* https://github.com/remarkjs/react-markdown/issues/565
* https://github.com/orgs/remarkjs/discussions/681
* https://github.com/orgs/remarkjs/discussions/1114
* https://github.com/orgs/remarkjs/discussions/674

Using search before re-posting a question can help you find an answer faster, and respects the time of others who have already taken the time to look into what happens and documented how `react-markdown` and `tailwind` can be combined.
The support guide offers some helpful suggestions on how to find existing answers and ask more effective questions https://github.com/remarkjs/.github/blob/main/support.md
